### PR TITLE
Vtouati pbi 298778 GitHub QA

### PR DESCRIPTION
**Pull request by VictoriaTouati-AVEVA**

### CODE REVIEW

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+uid: edit-transfer
+---
+
+# Edit a transfer
+
+Saved transfers can be edited by adding or removing AF elements and/or PI points.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:7]:
```suggestion
You can edit saved transfers by adding or removing AF elements and/or PI points.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+**Note:** Your transfer settings affect if streams and data get replicated or deleted from cloud storage after you have removed elements or explicit PI points from your transfer. See [Name the transfer and set data privacy settings](xref:transfer-data) for details.
+
+<!-- VTT: Add note about how the Opt-in setting (Transfer Settings window) affects streams/assets being deleted from cloud storage or replicated, depnding on the client's preference/option selection.-->
+
+Perform these steps to edit an existing transfer:
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:13]:
```suggestion
To edit an existing transfer, follow these steps:
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+1. In the `PI to OCS Agents` window, select the agent that contains the transfer you want to edit.
+
+1. Click the **View/Edit Transfer** button.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:17]:
```suggestion
1. Select **View/Edit Transfer**.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+1. In the transfer pane, select the **Edit Mode** toggle button.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:19]:
```suggestion
1. In the transfer pane, select the **Edit Mode** toggle.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+   Edit mode is enabled and the **Search** pane opens. 
+
+1. (Optional) To add PI points to a transfer, enter search criteria on the **PI Point Search** tab, click **Search**, select desired PI points, then click **Add PI Points to Transfer**.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:23]:
```suggestion
1. (Optional) To add PI points to a transfer, enter search criteria on the **PI Point Search** tab, select **Search**, select desired PI points, then select **Add PI Points to Transfer**. 
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+
+1. Click the **View/Edit Transfer** button.
+
+1. In the transfer pane, select the **Edit Mode** toggle button.
+
+   Edit mode is enabled and the **Search** pane opens. 
+
+1. (Optional) To add PI points to a transfer, enter search criteria on the **PI Point Search** tab, click **Search**, select desired PI points, then click **Add PI Points to Transfer**. 
+
+1. (Optional) To add AF elements to a transfer, enter search criteria on the **AF Elements Search** tab, click **Search**, select desired PI points, then click **Add AF Elements to Transfer**.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:25]:
```suggestion
1. (Optional) To add AF elements to a transfer, enter search criteria on the **AF Elements Search** tab, select **Search**, select desired PI points, and then select **Add AF Elements to Transfer**.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+
+1. (Optional) To remove PI points from a transfer, select the **PI Points** tab, select the PI points you want removed, select **Remove Points**, then select **Remove** to confirm.
+ 
+   **Note:** You can only remove explicit PI points from a transfer. To remove implicit PI points from a transfer, you will need to remove the associated AF element. 
+
+1. (Optional) Click **Undo Last Point Removal** to add the removed PI points back to the transfer.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:31]:
```suggestion
1. (Optional)  To add the removed PI points back to the transfer, select **Undo Last Point Removal**.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+1. (Optional) To remove AF elements from a transfer, select the **AF Elements** tab, select the AF elements you want removed, select **Remove Elements**, then select **Remove**.
+
+   The removed AF elements are marked as **Removed** in the **AF Elements** list. 
+
+   **Note:** Any implicitly referenced PI points that are not referenced by another AF element will also be removed from the transfer.
+
+1. (Optional) Click **Undo Last Element Removal** to add the removed PI points back to the transfer.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:39]:
```suggestion
1. (Optional) To add the removed PI points back to the transfer, select **Undo Last Element Removal** .
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+
+1. After transfer edits are done, click **Save** to retain these changes and return to the `PI to OCS Agents` window.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/edit-transfer.md:41]:
```suggestion
1. After transfer edits are done, select **Save** to retain these changes and return to the `PI to OCS Agents` window.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+The following table provides descriptions to the numbered elements in the Health Events window.
+
+| Number  | Description                                                  |
+| :-----: | ------------------------------------------------------------ |
+| 1  | The severity level of the log message. To filter by a particular severity level (Critical, Error, Warning, Information, Debug or Trace), click the filter icon and select one or more severity levels.        |
+| 2  | The source of an error message. To filter by source (agent, PI point indexing or AF indexing), click the filter icon and select one or more sources.  |
+| 3  | The date of the event. To filter log messages by a particular date and/or time, click the filter icon, then enter the date and or time in Month 00, 000 00:00:00 PM format.  |
+| 4  | The actual log message. To filter by a word or phrase, click the filter icon, then enter the word or phrase.      |
+| 5  | Select the **Clear All Events** button to remove all search filters. |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:19]:
```suggestion
| 5  | Select **Clear All Events** to remove all search filters. |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+| 6  | Select the **Export** button to save a copy of the health event messages to a .csv file.                               |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:20]:
```suggestion
| 6  | Select **Export** to save a copy of the health event messages to a .csv file.                               |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+
+The Health Events window provides information about an agent's activity and current status. Health events are based on log messages. You can sort messages by severity, source, time and message content.
+
+![](../../images/health-evts-window.png)
+
+The following table provides descriptions to the numbered elements in the Health Events window.
+
+| Number  | Description                                                  |
+| :-----: | ------------------------------------------------------------ |
+| 1  | The severity level of the log message. To filter by a particular severity level (Critical, Error, Warning, Information, Debug or Trace), click the filter icon and select one or more severity levels.        |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:15]:
```suggestion
| 1  | The severity level of the log message. To filter by a particular severity level (Critical, Error, Warning, Information, Debug or Trace), select the filter icon and select one or more severity levels.        |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+| 2  | The source of an error message. To filter by source (agent, PI point indexing or AF indexing), click the filter icon and select one or more sources.  |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:16]:
```suggestion
| 2  | The source of an error message. To filter by source (agent, PI point indexing or AF indexing), select the filter icon and select one or more sources.  |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 14]
```diff
@@ -0,0 +1,21 @@
...
+| 3  | The date of the event. To filter log messages by a particular date and/or time, click the filter icon, then enter the date and or time in Month 00, 000 00:00:00 PM format.  |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:17]:
```suggestion
| 3  | The date of the event. To filter log messages by a particular date and/or time, select the filter icon, then enter the date and/or time in `Month 00, 000 00:00:00 PM` format.  |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 15]
```diff
@@ -0,0 +1,21 @@
...
+| 4  | The actual log message. To filter by a word or phrase, click the filter icon, then enter the word or phrase.      |
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:18]:
```suggestion
| 4  | The actual log message. To filter by a word or phrase, select the filter icon, then enter the word or phrase.      |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 16]
```diff
@@ -265,18 +252,27 @@ To start a data transfer, follow these steps:
...
+
+   **Note:** The `Manage Agent` pane provides information about the agent associated with the transfer and the transfer progress.
+
+1. In the `Transfer Overview` section, view the transfer status as data is sent to the agent and stream data is created.
+
+   **Note:** The rate at which data transfers varies and depends on the density of data in the source PI Data Archive and/or AF server. See [Data transfer statuses](xref:data-transfer-statuses) for a list of transfer statuses and descriptions. To find out more information about an asset error, agent status, or asset create/update error, select the **Logs** button above the list of agents to access more information. Possible statuses that appear in the **Current Activity** field may indicate an issue and include Uncategorized Error, PI Point Type Change Detected, and No Valid PI Points In Transfer.   
 
-1. In the `Details` pane, expand the `Data Source` section to view the status of a transfer as data is sent.
+1. (Optional) To view more information about an agent's status, select the **Agent Health Events** button.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:267]:
```suggestion
1. (Optional) To view more information about an agent's status, select **Agent Health Events**.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 17]
```diff
@@ -265,18 +252,27 @@ To start a data transfer, follow these steps:
...
 
-   **Note:** The rate at which data transfers varies and depends on the density of data in the source PI Data Archive and/or AF server. See [Data transfer statuses](xref:data-transfer-statuses) for a list of transfer statuses and descriptions. To find out more information about an asset error, agent status, or asset create/update error, select the **Logs** button above the list of agents to access more information. Possible statuses that appear in the **Current Activity** field may indicate an issue include Uncategorized Error, PI Point Type Change Detected, and No Valid PI Points In Transfer.   
+   **Note:** See [Overview of the Health Events window](xref:health-evts-window) for more information.  
+ 
+1. (Optional) To see more information about log messages for the transfer, select the **Logs** button. See [Tenant log messages](xref:download-tenant-log) for more information.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:271]:
```suggestion
1. (Optional) To see more information about log messages for the transfer, select **Logs**. See [Tenant log messages](xref:download-tenant-log) for more information. 
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 18]
```diff
@@ -265,18 +252,27 @@ To start a data transfer, follow these steps:
...
+   **Note:** See [Overview of the Health Events window](xref:health-evts-window) for more information.  
+ 
+1. (Optional) To see more information about log messages for the transfer, select the **Logs** button. See [Tenant log messages](xref:download-tenant-log) for more information. 
 
-1. (Optional) If the fields in the `Details` pane indicate there are issues with the transfer or agent, select **Logs** to open the `Logs` window and view more information.
+1. (Optional) Click the `Transfer Metrics` tab to view transfer progress and metrics for stream and/or asset creation. See [Overview of transfer metrics](xref:overview-metrics) for more information.
```
Comment (afloresOSI) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:273]:
```suggestion
1. (Optional) To view transfer progress and metrics for stream and/or asset creation, select the `Transfer Metrics` tab. See [Overview of transfer metrics](xref:overview-metrics) for more information. 
```

**Review by afloresOSI — CHANGES_REQUESTED** [review]
Minor wording edits. I

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md [hunk 19]
```diff
@@ -0,0 +1,37 @@
...
+| Historical Start | The historical start date of the transfer. |
+| Historical End | The historical end date of the transfer. |
+| Total PI Points In Transfer | Total number of PI points selected for transfer. |
+| Total Points Updated | Total number of PI points updated during the transfer process. |
+| Stream Creation Status | The progress of streams creation during transfer progression.  |
+| Total Streams Created | Total number of streams created.  |
+| Total Streams Updated | Total number of streams updated with new configuration and metadata. |
+| Total Streams Deleted | Total number of deleted streams. |
+| Stream Creation Errors  | Total number of errors generated while creation streams. |
+| Stream Update Errors | Number of errors generated while updating streams. See the `Transfer Overview` section on the `Manage Agent` pane for error details. |
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:27]:
```suggestion
| Stream Update Errors | Number of errors generated while updating streams. See the `Transfer Overview` section on the **Manage Agent** tab for error details. |
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:27]:
```suggestion
| Stream Update Errors | Number of errors generated while updating streams. See the `Transfer Overview` section on the **Manage Agent** tab for error details. |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md [hunk 20]
```diff
@@ -0,0 +1,37 @@
...
+| Stream Deletion Errors | Number of errors generated during stream deletion. See the `Transfer Overview` section on the `Manage Agent` pane for error details. |
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:28]:
```suggestion
| Stream Deletion Errors | Number of errors generated during stream deletion. See the `Transfer Overview` section on the **Manage Agent** tab for error details. |
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:28]:
```suggestion
| Stream Deletion Errors | Number of errors generated during stream deletion. See the `Transfer Overview` section on the **Manage Agent** tab for error details. |
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md [hunk 21]
```diff
@@ -0,0 +1,37 @@
...
+| AF Server Version            | The version of Asset Framework (AF) that is installed on the source server configured to send AF element data via the agent.  |
+| Total Elements in Transfer | The number of AF elements configured in the transfer. |
+| Total Elements Transferred | The number of AF elements transferred to the PI to OCS Agent.  |
+| Assets Created Per Second | The number of assets created per second during the transfer. |
+| Total Assets Created | The total number of assets created. |
+| Assets Updated | The number of assets updated during the data transfer. |
+| Total Assets Deleted | The total number of deleted assets. |
+| Asset Create/Update Errors | The number of errors generated during asset creation or updating. |
+| Asset Deletion Errors | The number of errors generated during asset deletion. See the `Transfer Overview` section on the `Manage Agent` pane for error details.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:37]:
```suggestion
| Asset Deletion Errors | The number of errors generated during asset deletion. See the `Transfer Overview` section on the **Manage Agent** tab for error details.
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:37]:
```suggestion
| Asset Deletion Errors | The number of errors generated during asset deletion. See the `Transfer Overview` section on the **Manage Agent** tab for error details.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/data-transfer-statuses.md [hunk 22]
```diff
@@ -4,7 +4,7 @@ uid: data-transfer-statuses
 
 # Data transfer statuses
 
-The status and progress of a data transfer is shown in the Transfer Overview section of the Details pane. The following table lists these data statuses and their meanings.
+The status and progress of a data transfer is shown in the Transfer Overview section of the Manage Agent pane. The following table lists these data statuses and their meanings.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/data-transfer-statuses.md:7]:
```suggestion
The status and progress of a data transfer is shown in the `Transfer Overview` section of the **Manage Agent** tab. The following table lists these data statuses and their meanings.
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/data-transfer-statuses.md:7]:
```suggestion
The status and progress of a data transfer is shown in the `Transfer Overview` section of the **Manage Agent** tab. The following table lists these data statuses and their meanings.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 23]
```diff
@@ -0,0 +1,21 @@
+---
+uid: health-evts-window
+---
+
+# Overview of the Health Events window
+
+The Health Events window provides information about an agent's activity and current status. Health events are based on log messages. You can sort messages by severity, source, time and message content.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:7]:
```suggestion
The `Health Events` window provides information about an agent's activity and current status. Health events are based on log messages. You can sort messages by severity, source, time and message content.
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:7]:
```suggestion
The `Health Events` window provides information about an agent's activity and current status. Health events are based on log messages. You can sort messages by severity, source, time and message content.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md [hunk 24]
```diff
@@ -0,0 +1,21 @@
...
+
+![](../../images/health-evts-window.png)
+
+The following table provides descriptions to the numbered elements in the Health Events window.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:11]:
```suggestion
The following table provides descriptions to the numbered elements in the `Health Events` window.
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/health-events-window.md:11]:
```suggestion
The following table provides descriptions to the numbered elements in the `Health Events` window.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md [hunk 25]
```diff
@@ -0,0 +1,37 @@
+---
+uid: overview-metrics
+---
+
+# Overview of transfer metrics
+
+The `Transfer Metrics` pane displays details about a transfer's progress by server connection. Metrics for streaming and historical events and the progress of stream and asset creation are shown.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:7]:
```suggestion
The **Transfer Metrics** tab displays details about a transfer's progress by server connection. Metrics for streaming and historical events and the progress of stream and asset creation are shown. 
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:7]:
```suggestion
The **Transfer Metrics** tab displays details about a transfer's progress by server connection. Metrics for streaming and historical events and the progress of stream and asset creation are shown. 
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md [hunk 26]
```diff
@@ -0,0 +1,37 @@
+---
+uid: overview-metrics
+---
+
+# Overview of transfer metrics
+
+The `Transfer Metrics` pane displays details about a transfer's progress by server connection. Metrics for streaming and historical events and the progress of stream and asset creation are shown. 
+
+The following table provides a description of the fields in the `Transfer Metrics` pane.
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:9]:
```suggestion
The following table provides a description of the fields in the **Transfer Metrics** tab.
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/overview-metrics.md:9]:
```suggestion
The following table provides a description of the fields in the **Transfer Metrics** tab.
```

File: content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md [hunk 27]
```diff
@@ -29,7 +29,7 @@ Transfer creation consists of the following tasks:
 
 1. Initiate the data transfer and monitor its progress. 
 
-**Note:** If you have configured an AF server, you will not be able to create a transfer until AF indexing is complete. AF indexing status is listed on the `Details` pane in the `PI to OCS Agents` page. The following image shows AF indexing in progress:
+**Note:** If you have configured an AF server, you will not be able to create a transfer until AF indexing is complete. AF indexing status is listed on the `Manage Agent` pane in the `PI to OCS Agents` window. The following image shows AF indexing in progress:
```
Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:32]:
```suggestion
**Note:** If you have configured an AF server, you will not be able to create a transfer until AF indexing is complete. AF indexing status is listed on the **Manage Agent** tab in the `PI to OCS Agents` window. The following image shows AF indexing in progress:
```

Comment (jasnames) [inline content/add-organize-data/collect-data/connectors/pi-to-ocs/pi-to-ocs-data-transfers/transfer-data-to-ocs.md:32]:
```suggestion
**Note:** If you have configured an AF server, you will not be able to create a transfer until AF indexing is complete. AF indexing status is listed on the **Manage Agent** tab in the `PI to OCS Agents` window. The following image shows AF indexing in progress:
```

**Review by jasnames — CHANGES_REQUESTED** [review]
See suggestions. Other than what Angela already suggested, mostly just consistency of pane vs tab. Looks good!

**Review by afloresOSI — APPROVED** [review]

**Review by jasnames — APPROVED** [review]

**Review by jasnames — APPROVED** [review]

**Pull request merged**